Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Hathor Labs
+# This software is provided ‘as-is’, without any express or implied
+# warranty. In no event will the authors be held liable for any damages
+# arising from the use of this software.
+# This software cannot be redistributed unless explicitly agreed in writing with the authors.
+
+FROM node:14 AS builder
+
+COPY package.json /app/
+RUN cd /app && npm install --global --unsafe-perm tsdx@0.14.1
```

Comment:
Shouldn't we add the `tsdx` to the package.json?

Reply:
We already have it on `devDependencies`, I agree with you and removed the global tsdx from the dockerfile on b4fa93ab4ac3b7dd6eaf75d48df6a47d9879c08d

Reply:
Great.